matrix_content_scanner/utils/constants.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2022 The Matrix.org Foundation C.I.C.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from enum import Enum
+
+
+class ErrCode(Enum):
```

Comment:
```suggestion
class ErrCode(str, Enum):
```
I believe doing this means that a JSON encoder will have no trouble encoding it as a response

Reply:
str and repr won't help the JSON encoder AFAIK

Reply:
Weird, I remember them helping with that in my earlier testing but now that I'm testing it again I can't get it to work.